server/routes/testutils/appSetup.ts

```diff
@@ -49,7 +37,7 @@ function appSetup(services: Services, production: boolean, userSupplier: () => E
 export function appWithAllRoutes({
   production = false,
   services = {},
-  userSupplier = () => user,
+  userSupplier = () => null,
```

Comment:
This is just propping up a single test to show that the app responds properly with a not found page - a lot of stuff here isn't actually being used...